### PR TITLE
Add semi-auto script for the second release part

### PR DESCRIPTION
**Pull request by chanseokoh**
The script covers steps from 14 in the release instructions. I used it to make the last release, so I know it's working. It's semi-automatic and interactive, as some steps need to get some values such as a path to an Eclipse executable or a release version string. The script will certainly help a lot, since the steps are long, can be error-prone and time-consuming.

The script is safe (in terms of public visibility) to cancel out anytime and re-run, unless you execute the final step which makes things visible to public.

I don't think I follow the Google-style, if it exists. I won't mind if we don't merge this. For example, this could be kept in a separate branch.

Part of #354

### CODE REVIEW

File: release/release_after_kokoro.sh [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+echo "#"
+echo "#     $LOCAL_REPO/artifacts.xml"
+echo "#     $LOCAL_REPO/content.xml"
+echo "#     $LOCAL_REPO/features/*"
+echo "#     $LOCAL_REPO/plugins/*"
+echo "#"
+ask_proceed
+
+echo "The following steps require running an Eclipse binary."
+while true; do
```
Comment (elharo) [inline release/release_after_kokoro.sh]: I'm tempted to suggest that ECLIPSE_HOME should be set before running the script. Maybe if not set, default to the path on Goobuntu.

File: release/release_after_kokoro.sh [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+echo "#     $LOCAL_REPO/plugins/*"
+echo "#"
+echo "# into 'gs://cloud-tools-for-eclipse/<VERSION>/'."
+echo "#"
+ask_proceed
+
+while true; do
+    echo -n "Enter version: "
+    read VERSION
+    echo -n "Verify version: "
```
Comment (elharo) [inline release/release_after_kokoro.sh]: Is it really necessary to verify the version? Or maybe only verify the version if it looks off via regexp?

File: release/release_after_kokoro.sh (hunk 1 above)
Comment (chanseokoh) [inline release/release_after_kokoro.sh]: Yeah, failing fast seems better. The version of the bundled Eclipse on my Laptop is too old (3.8), so I'd rather require explicitly setting ECLIPSE_HOME.

File: release/release_after_kokoro.sh (hunk 2 above)
Comment (chanseokoh) [inline release/release_after_kokoro.sh]: Ok, reordring by putting `read VERSION` before `ask_proceed` seems sufficient.

**Pull request merged**